settings.php

```diff
@@ -1037,32 +1037,32 @@ function convertseconds($argument)
                                                     <div class="checkbox">
                                                         <label><input type="checkbox" name="whitelist" value="true"
-                                                                      checked>
-                                                            Whitelist</label>
+                                                                      checked><strong>
+                                                            Whitelist</strong></label>
                                                     </div>
                                                     <div class="checkbox">
                                                         <label><input type="checkbox" name="blacklist" value="true"
-                                                                      checked>
-                                                            Blacklist (exact)</label>
+                                                                      checked><strong>
+                                                            Blacklist (exact)</strong></label>
                                                     </div>
                                                     <div class="checkbox">
                                                         <label><input type="checkbox" name="wildlist" value="true"
-                                                                      checked>
-                                                            Blacklist (wildcard)</label>
+                                                                      checked><strong>
+                                                            Blacklist (wildcard)</strong></label>
                                                     </div>
                                                 </div>
                                             </div>
                                             <div class="col-lg-6 col-md-12">
                                                 <div class="form-group">
-                                                    <label for="zip_file">File input</label>
-                                                    <input type="file" name="zip_file" id="zip_file">
-                                                    <p class="help-block">Upload only Pi-hole backup files.</p>
+                                                    <p for="zip_file">Backup archive to upload:</p>
+                                                    <strong><input type="file" name="zip_file" id="zip_file"></strong>
+                                                    <p class="help-block">Upload only Pi-hole backup archives.</p>
                                                     <button type="submit" class="btn btn-default" name="action"
-                                                            value="in">Import
+                                                            value="in">Upload
```

Comment:
Keep this button as `Import` so that the user knows the uploaded data will be imported by clicking the button (instead of just uploaded).